package.json

```diff
@@ -71,8 +71,10 @@
     "react-router-dom": "^6.16.0",
     "react-scripts": "5.0.1",
     "react-secure-storage": "^1.3.0",
+    "react-transition-group": "^4.4.5",
     "stream-browserify": "^3.0.0",
     "typescript": "^4.9.5",
+    "use-window-focus": "^1.4.2",
```

Comment:
Could you please create own useWindowFocus hook in order not to increase the number of external libraries and to have this code in our database?

Also new libraries need to be added to the Credits list:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/c351b3b6-7f68-43e6-9d6a-bc14e27bd7f4)

Reply:
I removed the dependency and created a shared hook instead.

Thanks for the tip about the Credits screen, I'll keep that in mind! But for future reference, is the URL that "Credits" links to supposed to be public? When I click it, I only see the below. Should I have access to it?

<img width="609" alt="Screenshot 2024-02-05 at 4 53 45 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/9d28b07d-f948-43f2-94f8-109f6ed17f89">

Reply:
I discovered that I needed to open the Credits screen in an incognito window to have view access. I also found a [corresponding page](https://mindlogger.atlassian.net/wiki/x/t4BNF) in Confluence that seems to mirror what's on this help centre page. Is the Confluence page the source of truth and I can just add to it, or is there another process?

Reply:
Thank you for considering my comments. Regarding adding the library to the list, I think yes, after the merge of PR the new library needs to be added to the document.

Reply:
We have this document in the Confluence with the list of libraries:
https://mindlogger.atlassian.net/l/cp/pmQG5Uw3.
If you have access to it, please add the library to the list in this document after the merge.